plugins/zabbix/alerta_zabbix.py

```diff
@@ -45,11 +47,22 @@ def status_change(self, alert, status, text):
 
         event_id = alert.attributes.get('eventId', None)
         trigger_id = alert.attributes.get('triggerId', None)
+        more_info = alert.attributes.get('moreInfo',None)
```

Comment:
Unlikely to be in `moreInfo` if not in `eventId` so just return if not in `eventId`.